openwisp_controller/pki/migrations/0007_remove_ca_cert_fields.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.0.8 on 2018-09-26 11:26
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pki', '0006_add_x509_passphrase_field'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='ca',
+            name='passphrase',
+        ),
+        migrations.RemoveField(
+            model_name='cert',
+            name='passphrase',
```

Comment:
these fields should not be removed because it's a new feature of django-x509 which will be released soon, see the travis build for more info: https://github.com/openwisp/openwisp-controller/blob/master/.travis.yml#L46-L47